pkg/api/server/cel2sql/convert_test.go

```diff
@@ -18,234 +18,237 @@ import (
 	"errors"
 	"testing"
 
-	"github.com/tektoncd/results/pkg/api/server/cel"
-
+	"github.com/google/cel-go/cel"
 	"github.com/google/go-cmp/cmp"
+	resultspb "github.com/tektoncd/results/proto/v1alpha2/results_go_proto"
 )
 
-func TestConvertRecordExpressions(t *testing.T) {
+func newTestView() *View {
```

Comment:
Couldn't we use the actual view rather than building a new one for testing? Asking because by doing so, we might forget  to configure the actual view properly and do that only in the test one or vice-versa and ultimately, we might end up not exercising the actual interpreter's logic.